osu.Game/Overlays/Chat/ChatTabControl.cs

```diff
@@ -191,6 +199,43 @@ public ChannelTabItem(Channel value) : base(value)
                     }
                 };
             }
+
+            public class ChannelSelectorTabItem : ChannelTabItem
+            {
+                public override bool Active
+                {
+                    get { return base.Active; }
+                    set
+                    {
+                        activeBindable.Value = value;
+                        base.Active = value;
+                    }
+                }
+
+                private readonly Bindable<bool> activeBindable;
+
+                public ChannelSelectorTabItem(Channel value, Bindable<bool> active) : base(value)
+                {
+                    activeBindable = active;
+                    Depth = float.MaxValue;
+                    Width = 45;
+
+                    icon.Icon = 0;
+                    icon.X = 0;
+
+                    text.TextSize = 45;
+                    textBold.TextSize = 45;
+                }
+
+                [BackgroundDependencyLoader]
+                private new void load(OsuColour colour)
+                {
+                    backgroundInactive = colour.Gray2;
+                    backgroundActive = colour.Gray3;
+
+                    updateState();

```

Comment:
rather than calling updateState twice, maybe move it to `LoadComplete` of `ChannelTabItem`